conduit/data/config.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
-	"gopkg.in/yaml.v3"
+	yaml "gopkg.in/yaml.v3"
```

Comment:
It was failing my local `golangci-lint` but not entirely sure why it's ok in CI. Many other similar changes.